src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const costs = 40;
+  const rentalCosts = days * costs;
+
+  if (days <= 2) {
+    return rentalCosts;
+  } else if (days >= 3 && days < 7) {
+    return rentalCosts - 20;
+  } else if (days >= 7) {
+    return rentalCosts - 50;
+  }
```

Comment:
```suggestion
  if (days <= 2) {
    return rentalCosts;
  }
  
  if (days < 7) {
    return rentalCosts - 20;
  }
  
  return rentalCosts - 50;
```
Так как у тебя в каждом if есть return - else можно убрать
Повыноси числа(3,7,20,50) в переменные, и дай им названия, чтобы было понятно за что эта переменная отвечает